CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 
 # Changelog
 
+### #61 Story CORE-428 | CLI get installed brokers command
+- features:
+    - added a Brokers method and Brokers interface to controllers interface
+    - changed controller client to comply with new interface
+    - implemented a broker data interface (DI)
+    - changed insprd's server struct, http client, handler structure, and their mocks to include a BrokerManager
+    - created a handler to get broker data from insprd
+    - implemented cli command for getting broker data
+-  tests:
+    - unit tested handler
+    - interation test realized on cluster, flying colours
```

Comment:
integration*, and I think the "flying colours" is unnecessary for the changelog description